### PR TITLE
Add guide on how to remove clusters stuck in pending state.

### PR DESCRIPTION
**Pull request by makkes**
## Jira Ticket
no ticket

## Description of changes being made
Add guide on how to remove clusters stuck in pending state. See https://mesosphere.slack.com/archives/CJQURP1FT/p1603889592292200 for context.

## Checklist
- [x] Change all affected versions, if applicable (e.g. 1.13, 2.0, 2.1).
- [x] Test all commands and procedures, if applicable.
- [x] Create your PR against `staging`, not `master`. 
- [ ] Update all links if you are moving a page.
- [ ] Add release date to Release Notes page in the following format: <Package> was released on <Day>, <Month> <Year> ie `Mesosphere® DC/OS™ 2.1.0 was released on 9, June 2020`

See the [contribution guidelines](https://wiki.d2iq.com/display/ENG/Contributing+to+Docs) for more information.

### CODE REVIEW

File: pages/ksphere/kommander/1.1/troubleshooting/index.md [hunk 1]
```diff
@@ -59,3 +59,24 @@ If you cannot access a specific page, button, or action, it is likely that your
...
+
+`kubectl -n WORKSPACE_NAMESPACE get kommandercluster`
+
+Replace `WORKSPACE_NAMESPACE` with the actual namespace name of the workspace you're acting in (you will find this name by navigating to `https://YOUR_CLUSTER_DOMAIN_OR_IP_ADDRESS/ops/portal/kommander/ui/#/workspaces` in your browser.)
+
+Delete it by issuing this command:
+
+`kubectl -n WORKSPACE_NAMESPACE delete kommandercluster CLUSTER_NAME`
+
+If the resource does not go away after some minutes, you may have to remove its finalizers:
```
Comment (russurquhart1) [inline pages/ksphere/kommander/1.1/troubleshooting/index.md:77]:
```suggestion
1. If the resource does not go after a short time, remove its finalizers. Enter the following command:
```

File: pages/ksphere/kommander/1.1/troubleshooting/index.md [hunk 2]
```diff
@@ -59,3 +59,24 @@ If you cannot access a specific page, button, or action, it is likely that your
...
+
+`k -n WORKSPACE_NAMESPACE patch kommandercluster CLUSTER_NAME --type json -p '[{"op":"remove", "path":"/metadata/finalizers"}]'`
+
+After running that command, the cluster should vanish from the Kommander UI.
```
Comment (russurquhart1) [inline pages/ksphere/kommander/1.1/troubleshooting/index.md:81]:
```suggestion
This removes the cluster from the Kommander UI.
```

File: pages/ksphere/kommander/1.2/troubleshooting/index.md [hunk 3]
```diff
@@ -61,4 +61,24 @@ If you cannot access a specific page, button, or action, it is likely that your
 
 Similarly, to diagnose other missing actions for each resource listed above, the corresponding `update`, `delete`, and `create` permissions can be substituted in the place of `list` in the provided commands.
```
Comment (russurquhart1) [inline pages/ksphere/kommander/1.2/troubleshooting/index.md:62]:
```suggestion
Use the `update`, `delete`, and `create` permissions, like the `list` command is used, to diagnose other missing actions for each resource.
```

File: pages/ksphere/kommander/1.2/troubleshooting/index.md [hunk 4]
```diff
@@ -61,4 +61,24 @@ If you cannot access a specific page, button, or action, it is likely that your
...
 
+### I cannot detach an attached cluster that is "Pending"
```
Comment (russurquhart1) [inline pages/ksphere/kommander/1.2/troubleshooting/index.md:64]:
```suggestion
## I cannot detach an attached cluster that is "Pending"
```

File: pages/ksphere/kommander/1.2/troubleshooting/index.md (hunk 3 above)
Comment (russurquhart1) [reply pages/ksphere/kommander/1.2/troubleshooting/index.md:62]: Please use the same comments I gave for 1.1, for 1.2.

**Review by russurquhart1 — APPROVED** [review]
I gave comments for 1.1, please use those for 1.2, in the analogous locations.

**Review by samvantran — COMMENTED** [review]
Thanks Max! Good for me post-Russ comments

**Review by samvantran — APPROVED** [review]

**Pull request merged**